.github/workflows/test.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04]
+        os: [ubuntu-20.04, ubuntu-22.04, actuated-arm64-4cpu-12gb]
```

Comment:
GitHub App has to be installed from https://github.com/apps/self-actuated

Reply:
@caniszczyk @cyphar May I install this?

Reply:
@amye also has the necessary permissions IIRC. I can't install it unfortunately, but I've sent a request.

Reply:
Try it now, but "This GitHub App is for customers with an active Actuated subscription."

Reply:
Thanks, the app seems now installed